h11/_headers.py

```diff
@@ -132,11 +132,19 @@ def normalize_and_validate(headers, _parsed=False):
         raw_name = name
         name = name.lower()
         if name == b"content-length":
-            if saw_content_length:
-                raise LocalProtocolError("multiple Content-Length headers")
+            if b"," in value:
```

Comment:
Since `b"," in value` already does a full run of the string does it make sense to call `.split(b",")` then the set logic if `len(...) > 1`?

Reply:
I think so, I've changed it. Thanks